dev/com.ibm.ws.microprofile.config.1.1.services/src/com/ibm/ws/microprofile/config11/services/Config11ProviderResolverComponent.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.config11.services;
+
+import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
+import org.osgi.service.component.annotations.Component;
+
+import com.ibm.ws.container.service.state.ApplicationStateListener;
+import com.ibm.ws.microprofile.config.archaius.ConfigProviderResolverImpl;
+
+@Component(name = "com.ibm.ws.microprofile.config11.services.Config11ProviderResolverComponent", service = { ConfigProviderResolver.class,
+                                                                                                             ApplicationStateListener.class }, property = { "service.vendor=IBM" }, immediate = true)
+public class Config11ProviderResolverComponent extends ConfigProviderResolverImpl {}
```

Comment:
Special award for the silliest line-breaking!

If this is the formatter's fault, we should probably review the project formatting settings.

Reply:
I suspect it's configured to allow breaking between array literal elements but not between annotation parameters.